### ISSUE TITLE
background is all blank

### ISSUE THREAD

**Issue by takaakimasaki** [EBukin/devPTIpack#1]
When you download a map from the app, the background is now all blank. It would be nice to have a background from OpenStreetMap as a default. 

![somalia_flood](https://user-images.githubusercontent.com/13006617/140460397-45744520-f0d9-40e6-866f-4a1046ae0013.png)

**Comment by EBukin** [comment]
Yep... Can't fix it. I do not know what is the reason. It seems that there is a problem with the pantomJS and webshot packages. I found [this issue](https://github.com/wch/webshot/issues/107), but it does not seems that I there will be any help...

I propose to remove download "png" option leaving only HTML.

**Comment by EBukin** [comment]
Here is a [similar issue](https://github.com/jbkunst/highcharter/issues/702)...

**Comment by EBukin** [comment]
Wait a minute! R studio has just developed a new package [webshot2](https://rstudio.github.io/webshot2/index.html)! We will use it. It is working also on the secured laptop.

**Comment by EBukin** [comment]
After converting to `webshot2` , errors pops up silently in the logs about `chromote` package.

```
2021/11/07 16:22:34.259509185 ​​`google-chrome` and `chromium-browser` were not found. Try setting the CHROMOTE_CHROME environment variable or adding one of these executables to your PATH.
2021/11/07 16:22:34.262425035 Warning: Error in initialize: Invalid path to Chrome
```
Attempting the following workaround:
https://community.rstudio.com/t/how-to-properly-configure-google-chrome-on-shinyapps-io-because-of-webshot2/109020

**Comment by EBukin** [comment]
Linked issues

[https://github.com/wch/webshot/issues/72](https://github.com/wch/webshot/issues/72)
[https://github.com/jbkunst/highcharter/issues/702](https://github.com/jbkunst/highcharter/issues/702)

**Comment by EBukin** [comment]
[This](https://community.rstudio.com/t/how-to-properly-configure-google-chrome-on-shinyapps-io-because-of-webshot2/109020) workaround does not work. We shall try this one too: [https://github.com/RLesur/chrome_print_shiny](https://github.com/RLesur/chrome_print_shiny).

**Comment by EBukin** [comment]
I temporarily remove the png option.
